src/scene.js

```diff
@@ -343,6 +338,58 @@ export default class Scene {
     return hash[[r,g,b]];
   }
 
+  pickCustom(x, y, opt = {}) {
+    const gl = this.gl;
+
+    if (this.pickingFBO === undefined) {
+      this.pickingFBO = new Framebuffer(gl, {
+        width: gl.canvas.width,
```

Comment:
As I remember it, the picking buffer was previously quite small, we just a couple of pixels around the one we want to sample?